docs/shared/angular-tutorial/angular-monorepo.md

```diff
@@ -0,0 +1,1034 @@
+              },
+              {
+                "sourceTag": "type:ui",
+                "onlyDependOnLibsWithTags": ["type:ui"]
+              },
+              {
+                "sourceTag": "scope:orders",
+                "onlyDependOnLibsWithTags": [
+                  "scope:orders",
+                  "scope:products",
+                  "scope:shared"
+                ]
+              },
+              {
+                "sourceTag": "scope:products",
+                "onlyDependOnLibsWithTags": ["scope:products", "scope:shared"]
+              },
+              {
+                "sourceTag": "scope:shared",
+                "onlyDependOnLibsWithTags": ["scope:shared"]
+              }
+            ]
+          }
+        ]
+      }
+    },
+    ...
+  ]
+}
+```
+
+To test it, go to your `libs/products/src/lib/products.module.ts` file and import the `OrderListComponent` from the `orders` project:
+
+```ts {% fileName="libs/products/src/lib/products.module.ts" %}
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ProductListComponent } from './product-list/product-list.component';
+
+// This import is not allowed 👇
+import { OrderListComponent } from '@angular-monorepo/orders';
+
+@NgModule({
+  imports: [CommonModule],
+  declarations: [ProductListComponent],
+  exports: [ProductListComponent],
+})
+export class ProductsModule {}
+```
+
+If you lint your workspace you'll get an error now:
+
+```{% command="nx run-many -t lint" %}
+ >  NX   Running target lint for 7 projects
+    ✖  nx run products:lint
+       Linting "products"...
+
+       /Users/isaac/Documents/code/nx-recipes/angular-monorepo/libs/products/src/lib/products.module.ts
+         4:1   error    A project tagged with "scope:products" can only depend on libs tagged with "scope:products", "scope:shared"  @nx/enforce-module-boundaries
+         4:10  warning  'OrderListComponent' is defined but never used                                                               @typescript-eslint/no-unused-vars
+
+       ✖ 2 problems (1 error, 1 warning)
+
+       Lint warnings found in the listed files.
+
+       Lint errors found in the listed files.
+
+
+    ✔  nx run orders:lint (1s)
+    ✔  nx run angular-store:lint (1s)
+    ✔  nx run angular-store-e2e:lint (689ms)
+    ✔  nx run inventory-e2e:lint (690ms)
+    ✔  nx run inventory:lint (858ms)
+    ✔  nx run shared-ui:lint (769ms)
+
+ ———————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+
+ >  NX   Ran target lint for 7 projects (3s)
+
+    ✔    6/7 succeeded [0 read from cache]
+
+    ✖    1/7 targets failed, including the following:
+         - nx run products:lint
+```
+
+If you have the ESLint plugin installed in your IDE you should immediately see an error:
+
+![ESLint module boundary error](/shared/images/tutorial-angular-standalone/angular-standalone-module-boundaries.png)
```

Comment:
this image doesn't seem to load for some reason. seems to be from the angular-standalone tutorial. That shouldn't be a problem, but maybe we should move it to some shared location so it doesn't get deleted sometime in the future by accident.